src/components/WhoAreWe.astro

```diff
@@ -0,0 +1,29 @@
+<div class="mx-auto max-w-lg">
```

Comment:
Take a look at tailwind's columns specifiers, I think we can use either a `md` or `lg` breakpoint to switch to 2 columns. (Also I think max-w-lg is unnecessary after we do that?)

https://tailwindcss.com/docs/columns

Reply:
Ok let's do the breakpoint stuff in a future change